### PR TITLE
[W5] [T11-1] Windrich

### PR DESCRIPTION
**Pull request by windrichie**
Code in new class command adapted from existing command (AddCommand and ListCommand).

### CODE REVIEW

**Comment by JKupoluyi** [comment]
Good Job!

File: src/seedu/addressbook/commands/AddListCommand.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    public AddListCommand(Person toAdd) {
+        this.toAdd = toAdd;
+    }
+
+    public ReadOnlyPerson getPerson() {
+        return toAdd;
+    }
+
+    @Override
```
Comment (stephlewyh) [inline src/seedu/addressbook/commands/AddListCommand.java:65]: Missing header comment. All non-trivial methods should have java doc format header comments.

File: src/seedu/addressbook/commands/AddListCommand.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    public CommandResult execute() {
+        try {
+            addressBook.addPerson(toAdd);
+            List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+            return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+//            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
```
Comment (stephlewyh) [inline src/seedu/addressbook/commands/AddListCommand.java:71]: This is an unnecessary inline comment.

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -135,6 +139,32 @@ private Command prepareAdd(String args) {
         }
     }
 
+    private Command prepareAddList(String args) {
```
Comment (stephlewyh) [inline src/seedu/addressbook/parser/Parser.java:142]: Missing header comment. All non-trivial methods should have java doc format header comments.

**Review by stephlewyh — COMMENTED** [review]
Hi Windrich, 

Excellent job for a first attempt. Do note that you need to provide header comments for your enhancements and with non-trivial methods. Avoid leaving behind commented out code in your submission. 

You have provided tests with edge cases. Good job!
